docs/guides/getting-started/setup/vite.mdx

```diff
@@ -94,6 +94,12 @@ export default defineConfig({
 })
 ```
 
+:::info
+
+Note that if you are not using vanilla JavaScript, you should modify this config and add the necessary plugins your framework requires.
```

Comment:
@NathanaelRea What do you think about something like this, since create-vite's templates already contain the plugins?
```
Note that if you are not using vanilla JavaScript, you must keep the framework-specific plugins that were already in this file.
```
cc @lorenzolewis because i'm not happy with my own suggestion either 😂

Reply:
I mainly wanted to make it clear that the vite.config.ts given can be modified. It says "update the file with the following", but then it gives the entire file, which might lead the user to just overwrite their file with the one provided.

Reply:
Yeah we're on the same page